azurerm/internal/acceptance/steps.go

```diff
@@ -92,3 +92,10 @@ func (td TestData) RequiresImportErrorStep(configBuilder func(data TestData) str
 		ExpectError: RequiresImportError(td.ResourceType),
 	}
 }
+
+func (td TestData) RequiresImportErrorStepWithConfig(config string) resource.TestStep {
+	return resource.TestStep{
+		Config:      config,
+		ExpectError: RequiresImportError(td.ResourceType),
+	}
+}
```

Comment:
which means we should be able to remove this - since we should be encouraging leaning on `data` where possible (that means we could replay those tests easier in the future)